src/data-model/PropertyValueRelation.ts

```diff
@@ -1,7 +1,7 @@
 enum PropertyValueRelation {
```

Comment:
Could you add a comment here that these are also used as part of message keys and so any changes made to this enum have to also be reflected in `en.json` and `qqq.json`

Reply:
ok!